### ISSUE TITLE
IOS 15 users have problem loading or clicking any page when we have this plugins enabled in the website

### ISSUE THREAD

**Issue by josephmiller2000** [theajack/disable-devtool#14]
disable-devtool-auto
--
  | src='https://cdn.jsdelivr.net/npm/disable-devtool/disable-devtool.min.js'
  | md5='/////////removed/////'
  | tk-name='dt'
  | interval='1500'
  | disable-menu='true'
  | ></script>

I did use this version of JS to disable dev tool in my chat website. For some reason, all IOS 15 users with safari browser and chrome browser had issue loading the page and can't navigate to any page

Or even opening the URL in direct "_blank" also creating an issue closing the tab.

Please take a look at it and fix it.

**Comment by milindpatel63** [comment]
Although, not a fix, but here's a code to detect iOS via javascript...you can use this to disable the script on IOS specifically

```
function iOS() {
  return [
    'iPad Simulator',
    'iPhone Simulator',
    'iPod Simulator',
    'iPad',
    'iPhone',
    'iPod'
  ].includes(navigator.platform)
  // iPad on iOS 13 detection
  || (navigator.userAgent.includes("Mac") && "ontouchend" in document)
}
```

**Comment by theajack** [comment]
Fixed! Please upgrade to Version 0.1.7.

**Comment by milindpatel63** [comment]
This issue still persists in Chrome on iOS.
Its fixed in safari on iOS, but still having this issue on chrome.
